ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -10,6 +10,11 @@
 from ocs_ci.ocs.resources.csv import CSV
 from ocs_ci.ocs.resources.packagemanifest import get_selector_for_ocs_operator, PackageManifest
 from ocs_ci.utility import utils
+from tests.helpers import check_local_volume, check_pvs_created
```

Comment:
not a good practice to import from tests dir into this file. Import is done the other way around.
Can you please move the implementation of these functions to another place under ocs_ci dir?

Reply:
Should I just add them in the same file? 
At the bottom?

Reply:
Pushed them to the bottom